.env.example

```
@@ -9,6 +9,14 @@ VITE_APPWRITE_COLLECTION_SESSIONS=sessions
 VITE_APPWRITE_COLLECTION_ACTIVITY_LOGS=activityLogs
 VITE_APPWRITE_COLLECTION_REPORTS=reports
 VITE_APPWRITE_COLLECTION_SETTINGS=settings
+VITE_APPWRITE_COLLECTION_HACKATHONS=hackathons
+VITE_APPWRITE_COLLECTION_HACKATHON_PARTICIPANTS=hackathonParticipants
+VITE_SONAR_WEBSITE_URL=https://sonar.knurdz.org
+VITE_SONAR_INVITE_SECRET=replace-with-a-long-random-secret-for-team-invites
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SONAR_INVITE_SECRET key should go before the VITE_SONAR_WEBSITE_URL key

(UnorderedKey)